From c5c5f67f5d5ef2b4f7145f6586a8b3d8dba7fab5 Mon Sep 17 00:00:00 2001
From: Ben Cooksley <bcooksley@kde.org>
Date: Mon, 7 Feb 2022 06:12:40 +1300
Subject: [PATCH] Purge all references to download.kde.org, replacing them with
 the correct endpoint of autoconfig.kde.org.

CCMAIL: plasma-devel@kde.org
CCMAIL: distributions@kde.org
(cherry picked from commit a428630ea4a46e967de71711c8da5cbaeb9b690b)
---
 src/kcmkwin/kwineffects/kwineffect.knsrc                 | 2 +-
 src/kcmkwin/kwinscripts/kwinscripts.knsrc                | 2 +-
 src/kcmkwin/kwintabbox/kwinswitcher.knsrc                | 2 +-
 src/plugins/kdecorations/aurorae/src/aurorae.knsrc.cmake | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/kcmkwin/kwineffects/kwineffect.knsrc b/src/kcmkwin/kwineffects/kwineffect.knsrc
index 6abcfb34f8..22bd1c98ec 100644
--- a/src/kcmkwin/kwineffects/kwineffect.knsrc
+++ b/src/kcmkwin/kwineffects/kwineffect.knsrc
@@ -45,7 +45,7 @@ Name[x-test]=xxWindow Manager Effectsxx
 Name[zh_CN]=窗口管理器特效
 Name[zh_TW]=視窗管理員效果
 
-ProvidersUrl=https://download.kde.org/ocs/providers.xml
+ProvidersUrl=https://autoconfig.kde.org/ocs/providers.xml
 Categories=KWin Effects
 StandardResource=tmp
 Uncompress=kpackage
diff --git a/src/kcmkwin/kwinscripts/kwinscripts.knsrc b/src/kcmkwin/kwinscripts/kwinscripts.knsrc
index 44faa872ba..c002f0eec5 100644
--- a/src/kcmkwin/kwinscripts/kwinscripts.knsrc
+++ b/src/kcmkwin/kwinscripts/kwinscripts.knsrc
@@ -44,7 +44,7 @@ Name[x-test]=xxWindow Manager Scriptsxx
 Name[zh_CN]=窗口管理器脚本
 Name[zh_TW]=視窗管理員指令稿
 
-ProvidersUrl=https://download.kde.org/ocs/providers.xml
+ProvidersUrl=https://autoconfig.kde.org/ocs/providers.xml
 Categories=KWin Scripts
 StandardResource=tmp
 Uncompress=kpackage
diff --git a/src/kcmkwin/kwintabbox/kwinswitcher.knsrc b/src/kcmkwin/kwintabbox/kwinswitcher.knsrc
index bef9bc07ea..d0f5c3b520 100644
--- a/src/kcmkwin/kwintabbox/kwinswitcher.knsrc
+++ b/src/kcmkwin/kwintabbox/kwinswitcher.knsrc
@@ -45,7 +45,7 @@ Name[x-test]=xxWindow Manager Switching Layoutsxx
 Name[zh_CN]=窗口管理器切换布局
 Name[zh_TW]=視窗切換器佈局
 
-ProvidersUrl=https://download.kde.org/ocs/providers.xml
+ProvidersUrl=https://autoconfig.kde.org/ocs/providers.xml
 Categories=KWin Switching Layouts
 StandardResource=tmp
 Uncompress=kpackage
diff --git a/src/plugins/kdecorations/aurorae/src/aurorae.knsrc.cmake b/src/plugins/kdecorations/aurorae/src/aurorae.knsrc.cmake
index a773c28c52..d0c7748852 100644
--- a/src/plugins/kdecorations/aurorae/src/aurorae.knsrc.cmake
+++ b/src/plugins/kdecorations/aurorae/src/aurorae.knsrc.cmake
@@ -40,7 +40,7 @@ Name[x-test]=xxAurorae Window Decorationsxx
 Name[zh_CN]=Aurorae 极光窗口装饰
 Name[zh_TW]=Aurorae 視窗裝飾
 
-ProvidersUrl=https://download.kde.org/ocs/providers.xml
+ProvidersUrl=https://autoconfig.kde.org/ocs/providers.xml
 Categories=Window Decoration Aurorae
 Uncompress=archive
 TargetDir=aurorae/themes
-- 
GitLab

